docs/mcd.md

```diff
@@ -0,0 +1,329 @@
+| reg-exp | Regular Expression | No | | |
+
+### `<boolean-options>`
+
+A required child of field when value-type is `boolean`, this is the container element for the two boolean options. It has no XML attributes. 
+
+### `<false-value>`
+
+A required child of boolean-options when value-type is `boolean`. This is the value to send to ConnectionBuilder() when the user does not check the box.
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| value | The value to send to the ConnectionBuilder() | No | Any string value | |
+
+### `<true-value>`
+
+A required child of boolean-options when value-type is `boolean`. This is the value to send to ConnectionBuilder() when the user checks the box.
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| value | The value to send to the ConnectionBuilder() | No | Any string value | |
+
+### `<selection-group>`
+
+A required child of field when value-type is `option`. This is the container element for the options that will appear in the drop-down. It has no XML attributes. 
+
+Multiple selection-group elements can be used; see `<conditions>`.
+
+### `<option>`
+
+A child of selection-group, this represents one entry in the drop-down. It has the following XML attributes. 
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| value | The value sent to ConnectionBuilder() | No | | |
+| label | The text the user sees | No | | |
+
+### `<conditions>`
+
+An optional child of field or selection group, this is used for conditional display. It specifies whether the field or selection group should be visible in the dialog based on the values the user has specified for other fields. 
+
+Multiple condition elements can be used, and will be OR'd. That is, the field or selection group will be visible when any one of the `<condition>` elements is matched.
+
+Hence, this is a container element for the condition elements. It has no XML attributes. 
+
+### `<condition>`
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| field | Name of the field whose value will be checked for equality | No | | |
+| value | The value to match. If the field's value is equal to this, the condition is true. | No | | |
+
+
+## Example
+
+The Connection Fields file below demonstrates both conditional display of fields and a non-editable field. 
+
+The images below show the Connection Dialog produced using this file. The left side shows the dialog as it is initially displayed, and the right side shows what it changes to when the user selects Authentication Kerberos. 
+
+<p float="left">
+  <img src="{{ site.baseurl }}/assets/mcd-connection-dialog-1.png" alt="Connection Dialog as initially displayed" align=top hspace=40/>
```

Comment:
For some reason cannot see the two images

Reply:
I'm assuming that the right thing will happen with {{ site.baseurl }} when this is merged and the result is published. I just copied the pattern from other files in the docs folder that references images in the assets folder.

Reply:
Ah okkay. Yes in that case it should happen